docs/node-port-local.md

```diff
@@ -31,6 +31,8 @@ NodePortLocal was introduced in v0.13 as an alpha feature, and was graduated to
 beta in v1.4, at which time it was enabled by default. Prior to v1.4, a feature
 gate, `NodePortLocal`, must be enabled on the antrea-agent for the feature to
 work. Starting from Antrea v1.7, NPL is supported on the Windows antrea-agent.
+From Antrea v1.14, NPL is GA. To enable this feature, you need to ensure that the
+`nodePortLocal.enable` flag is set to true in Antrea Agent configuration.
```

Comment:
It seems duplicate with the following section. It has mentioned `nodePortLocal.enable`, perhaps just say it's GA here?

Reply:
Agree, removed the "To enable this" sentence.